deploy/README.md

```diff
@@ -0,0 +1,228 @@
+# Scripts to deploy the whole environement and the application using bash
+
+Those steps allow you to deploy the whole environement from your local machine using Docker, AKS, ACR and MongoDB.
```

Comment:
environment.  I'd also at a minimum like to reference the automated HOL solution here in this document: https://microsoft.github.io/PartsUnlimitedMRPmicro/hols/circleci.html